server/channels/app/user.go

```diff
@@ -318,6 +318,17 @@ func (a *App) createUserOrGuest(c request.CTX, user *model.User, guest bool) (*m
 		}(ruser.Id)
 	}
 
+	userLimits, appErr := a.GetUserLimits()
+	if appErr != nil {
+		// we don't want to break the create user flow just because of this.
+		// So, we log the error, not return
+		mlog.Error("Error fetching user limits in createUserOrGuest", mlog.Err(appErr))
+	} else {
+		if userLimits.ActiveUserCount > userLimits.MaxUsersLimit {
+			mlog.Warn("User count has exceeded the permissible limit of %d users", mlog.Int("user_limit", userLimits.MaxUsersLimit))
+		}
+	}
```

Comment:
not a blocker but can this be improved instead of if inside of else?

Reply:
Yes, everything comes to this function - signup, invite, SSO, everything.

Reply:
How about the path to activate a user? Deactivate user -> Active user

Reply:
@marianunez forgot to comment, but I added check for this.